### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Application Graph_Color Contrast: Color contrast ratio for Tab items & links (blue color text) is 3.6:1 which is < 4.5:1

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3594]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ColorContrast; #MAS1.4.3; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated**: 1.4.3

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till Create New Project button and press enter. Navigate to Azure Spark/HDInsight-->Spark Project Scala and press Next button and fill all the details to create a new project
3)Tab till Azure Spark/HDInsight-->Spark Project Scala
4)Tab till Build Tools-->Maven
5)Tab till Next button and press enter
6)Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
7)Tab till HDInsight and link a cluster
8)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
9)Navigate to New-->Scala Class and press enter
10)Tab till object from menu and select it using keyboard
11)Override the code and Navigate to project name-->src-->main-->scala-->Project Name and right click on it & Tab till Submit Spark Application and press enter
12)Fill all the required details & Tab till finish button and press enter
13)Navigate to jobs under the  cluster (which we have used) and press enter
14)Verify the color contrast ratio for tab items(Application Graph, Stage Summary, Executors, Task Summary, Logs) & links(Open spark history UI, open yarn UI)

**Actual Result:**
 Color contrast ratio for Tab items & links (blue color text) is 3.6:1 which is < 4.5:1

**Expected Result:**
 Color contrast ratio for Tab items & links (blue color text) should be >=  4.5:1

**User Impact:**
Low vision users will find difficult to read the content if the text does not meet the standard color contrast ratio

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![Bug_3594_color contrast](https://user-images.githubusercontent.com/42638450/65420156-0ffaf180-de1e-11e9-8d93-aa0630233ba3.png)

**Comment by t-rufang** [comment]
The job view page is wrote with the help of JavaFX which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749697
